packages/app/src/specs/InlineRunAllSpecs.vue

```diff
@@ -6,6 +6,7 @@
     <button
       class="flex h-full w-full items-center justify-center"
       data-cy="run-all-specs-button"
+      data-run-all-specs-button
```

Comment:
In most places we use the `data-cy="..."` pattern - any reason you're just using a data attribute here?

Reply:
Oh I can pull this out now, removed. For a second I was using this in one of the higher level listeners to avoid reacting to events fired on this button element, before I realized we didn't need those listeners themselves.